### PR TITLE
Update slack-channel.md

### PR DESCRIPTION
**Pull request by reese-lee**
Updates this to include specific instructions on how to access our Slack channel ([per this feedback](https://github.com/open-telemetry/opentelemetry.io/issues/5665)), and remove old info about the previous channel.

### CODE REVIEW

File: content/en/community/end-user/slack-channel.md [hunk 1]
```diff
@@ -6,21 +6,17 @@ description: >-
...
 ---
 
-Previously, we had a private channel to provide a space for end users to discuss
-adoption and implementation issues while allowing them to maintain a layer of
-privacy and to meet compliance concerns. Now that the project and community have
-grown, and with the transition of the End User WG to End User SIG, we believe
-that this private channel is no longer needed.
+Join our Slack channel to connect with other OpenTelemetry end users! First, 
+[request an invitation](https://communityinviter.com/apps/cloud-native/cncf) 
+to CNCF's Slack instance.
```
Comment (tiffany76) [inline content/en/community/end-user/slack-channel.md:12]:
```suggestion
Join our Slack channel to connect with other OpenTelemetry end users! 

First, [request an invitation](https://communityinviter.com/apps/cloud-native/cncf) 
to CNCF's Slack instance. Next, join [`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ), and introduce yourself if you'd like. 
```

File: content/en/community/end-user/slack-channel.md [hunk 2]
```diff
@@ -6,21 +6,17 @@ description: >-
...
 
-As of April 2024, we will begin shifting all of our communications to the
-[`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ)
-channel (previously named the `#otel-user-research` channel), and will be
-archiving the private channel by **April 30, 2024**.
+Next, join [`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ),
+and introduce yourself if you would like.
```
Comment (tiffany76) [inline content/en/community/end-user/slack-channel.md:14]:
```suggestion
```

File: content/en/community/end-user/slack-channel.md [hunk 3]
```diff
@@ -6,21 +6,17 @@ description: >-
...
+Next, join [`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ),
+and introduce yourself if you would like. 
 
-We encourage you to join us over at
-[`#otel-sig-end-user`](https://cloud-native.slack.com/archives/C01RT3MSWGZ) to
-learn about our new charter, what you can expect from us as an end user, and
-more. Note the following:
+We encourage questions and discussions about adoption and implementation. 
+You can also hear about upcoming events, and get a chance to contribute
+to the project by participating in surveys and/or user feedback sessions.
```
Comment (tiffany76) [inline content/en/community/end-user/slack-channel.md:18]:
```suggestion
You can also hear about upcoming events and get a chance to contribute
to the project by participating in surveys and user feedback sessions. 
```

**Review by tiffany76 — APPROVED** [review]
Just a few copyedits. Thanks for taking care of this so quickly.

File: content/en/community/end-user/slack-channel.md (hunk 1 above)
Comment (reese-lee) [reply content/en/community/end-user/slack-channel.md]: I messed up and wasn't able to directly commit this suggestion, but have added it manually to the file.

**Review by avillela — APPROVED** [review]
Looks great!

**Review by cartermp — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

**Pull request merged**